esrally/telemetry.py

```diff
@@ -238,18 +240,19 @@ def java_opts(self, log_file):
             java_opts.append("-XX:+FlightRecorder")
             java_opts.append("-XX:FlightRecorderOptions=disk=true,maxage=0s,maxsize=0,dumponexit=true,dumponexitpath={}".format(log_file))
             jfr_cmd = "-XX:StartFlightRecording=defaultrecording=true"
-            if recording_template:
-                self.logger.info("jfr: Using recording template [%s].", recording_template)
-                jfr_cmd += ",settings={}".format(recording_template)
-            else:
-                self.logger.info("jfr: Using default recording template.")
         else:
             jfr_cmd += "-XX:StartFlightRecording=maxsize=0,maxage=0s,disk=true,dumponexit=true,filename={}".format(log_file)
-            if recording_template:
-                self.logger.info("jfr: Using recording template [%s].", recording_template)
-                jfr_cmd += ",settings={}".format(recording_template)
-            else:
-                self.logger.info("jfr: Using default recording template.")
+        if delay:
```

Comment:
When a `if` statement is directly below an existing if/elif/else statement, please add a newline to clarify that this is a different statement. This applies to duration and recording_template below.

Reply:
This has been updated in the latest commit.